### ISSUE TITLE
web components

### ISSUE THREAD

**Issue by samccone** [MarionetteLabs/Sesame.js#1]
I think this should be a seamless layer that we should keep in mind.

thoughts?

**Comment by jasonLaster** [comment]
Agreed, I've always liked the Backbone Api for this, but i'd love to hear more ideas.

``` js
Clock = View.extend({

  tag: 'clock',
  attributes: function() {

    return {time: Time.now()}
  }
});

AnalogClock = Clock.extend({ 

   attributes: function() {
       var attrs = this.super();
       return {
           hand: ':clap:'
       }
  }
});

```

**Comment by jamesplease** [comment]
I suspect we might be able to draw inspiration from Ember's implementation of [components](http://emberjs.com/api/classes/Ember.Component.html), which are like views but completely isolated from the outside world.

Aside from isolation, I agree with @jasonLaster that Backbone's API handles Web Components pretty well.

**Comment by jridgewell** [comment]
Re: https://github.com/babel-plugins/babel-plugin-incremental-dom/issues/9 and https://github.com/marionettejs/backbone.marionette/issues/2734
